### DISCUSSION TITLE
Does not drowing 2 chanel when audio is to short

### DISCUSSION THREAD

**Discussion by StanislavMelnychek** [katspaugh/wavesurfer.js#3699] [Q&A]
## Bug description
If you use a very short audio it only draws one channel
Could somebody help with this please, spend a lot of time on it already

## Environment
 - Browser: Chrome

## Minimal code snippet
```
import WaveSurfer from 'wavesurfer.js'

const wavesurfer = WaveSurfer.create({
  container: document.body,
  waveColor: 'rgb(200, 0, 200)',
  progressColor: 'rgb(100, 0, 100)',
  splitChannels: [{}, {}],
  url: 'https://upload.wikimedia.org/wikipedia/commons/1/11/H_is_for_horse.ogg',
})

wavesurfer.on('click', () => {
  wavesurfer.play()
})
```

## Screenshots

![wavesurfer-js-audio-waveform-player-JavaScript-library](https://github.com/katspaugh/wavesurfer.js/assets/137765025/5a816ae5-087e-40f3-a1c1-0e79afca3847)

**Comment by katspaugh** [comment]
The file you linked is a **mono** audio track.

**Comment by katspaugh** [comment]
Thanks for your sponsorship! 🙏
